sql/item_latents.sql

```diff
@@ -2769,4 +2769,16 @@ INSERT INTO `item_latents` VALUES(15532, 369, -3, 56, 0); -- MP Drain -3MP/tic
 -- Shadow Mantle/Ring
 -- -------------------------------------------------------
 INSERT INTO `item_latents` VALUES(13658, 10, 20, 32, 0); -- Darksday: VIT+20
-INSERT INTO `item_latents` VALUES(14646, 29, 10, 32, 0); -- Darksday: MDB+10
\ No newline at end of file
+INSERT INTO `item_latents` VALUES(14646, 29, 10, 32, 0); -- Darksday: MDB+10
+
+-- -------------------------------------------------------
+-- LAVALIER
+-- -------------------------------------------------------
+-- INSERT INTO `item_latents` VALUES (10961,CHARMED:LIGHT RESIST,-40,0,0);
```

Comment:
I mentioned for you to put this here because you can implement these right away. It should be...
    (10961, 60, -40, 13, 14)
for Lightres -40, latent condition being has status effect, param is 14 for effectid for charm status.